ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -201,6 +188,25 @@ protected override async Task ValidateAndSave()
             }
         }
 
+        private void AddPostMortemErrorsIfApplicable()
+        {
+            // Only validate post mortem if a notified case is changing to post mortem, or draft is post mortem
+            if (ClinicalDetails.IsPostMortem == true
+                && (Notification.NotificationStatus == NotificationStatus.Notified && Notification.ClinicalDetails.IsPostMortem != true)
+                || Notification.NotificationStatus == NotificationStatus.Draft)
```

Comment:
This code is only correct if `||` takes precedence over `&&`. This may be true (I can never remember!) but I would always add brackets to make the intention crystal clear - i.e. `A && (B || C)`